tox.ini

```diff
@@ -29,3 +29,5 @@ commands =
     coverage run --source ocean_provider -m py.test -s --basetemp={envtmpdir}
     coverage report
     coverage xml
+    ls -l .
```

Comment:
should we keep this? It looks like debugging code.

Reply:
removed